app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -275,23 +275,9 @@ class StudentForm extends React.Component {
     * Validates fields for student form fields
     */
   async validateStudentFields() {
-    var reject = (response) => { this.createStripeCustomer(response) };
+    var reject = (response) => { this.validateAddress(response) };
     var resolve = (response) => { this.createStripeCustomer({}) };
```

Comment:
do we need to do validateAddress too if there's no validation errors for the student?